client/me/notification-settings/settings-form/actions.jsx

```diff
@@ -35,11 +35,11 @@ class NotificationSettingsFormActions extends React.PureComponent {
 						onClick={ this.props.onSaveToAll }
 						isPrimary={ false }
 					>
-						{ this.props.translate( 'Save to All Sites' ) }
+						{ this.props.translate( 'Save to all sites' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.